pages/[resourceType]/[id]/care-gaps.tsx

```diff
@@ -262,7 +376,7 @@ const CareGapsPage = () => {
                           colorScheme="dark"
                           style={{ maxWidth: "77vw", height: "80vh", backgroundColor: "#FFFFFF" }}
```

Comment:
<img width="1495" alt="image" src="https://user-images.githubusercontent.com/30158156/183453129-5871d695-f64c-4fa1-8a1c-a664dcc57f6f.png">


Not sure if this is even something you would want to change but I figured that it was worth pointing out! I noticed that the care gaps input and prism component don't quite line up if the window is either resized or the request preview is pretty long (i.e. includes both an organization or practitioner). Very nit-picky UI thing I found so very understandable if this isn't a priority!

Reply:
I adjusted some padding so the alignment is a little better but it's still not great if the user resizes the page. I'll look into making it more dynamic for this and evaluate measure later in the week once I finish up search pagination stuff, if that's ok?

Reply:
Of course! It really isn't a huge deal at all and I don't even mind it I just thought I would point it out.